web/blackjack/src/app/game/game/game.component.ts

```diff
@@ -47,6 +47,10 @@ export class GameComponent implements OnInit {
     });
```

Comment:
You can save a reference to the subscription returned here, i.e.:
```
this.intervalSubscription = interval(500).subscribe(() => {
      this.fetchGame();
});
```

And then in ngOnDestroy you can unsubscribe.